airbyte-cdk/python/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
-## 0.11.2
+## 0.12.2
+Revert breaking change on `read_config` while keeping the improvement on the error message
+
```

Comment:
There seems to be a changelog missing here https://github.com/airbytehq/airbyte/commit/23b35ba27c7eca2859373b2b2cc79bbaad21a943